ehrql/tables/beta/tpp.py

```diff
@@ -459,3 +460,48 @@ class isaric_raw(EventFrame):
         datetime.date,
         description="Outcome date.",
     )
+
+
+@table
+class open_prompt(EventFrame):
+    """
+    Participants answer questions in questionnaires. Their answers are responses. There
+    is one questionnaire that we expect to be administered once, on day 0. There are
+    five questionnaires that we expect to be administered four times, on days 0, 30, 60,
+    and 90.
+
+    * Baseline questionnaire (administered once, on day 0)
+    * Covid and vaccination questionnaire
+    * Eq-5D questionnaire
+    * Productivity questionnaire
+    * Fatigue questionnaire
+    * Breathing problems questionnaire
+
+    Each time is a survey. That is, a survey comprises the responses to the questions in
+    between five and six questionnaires.
+    """
+
+    coded_event_id = Series(
+        CTV3Code,
+        description="[TBC] The question, as a CTV3 code",
+    )
+    snomedct_concept_id = Series(
```

Comment:
We've tended to use the phrase "code" rather than "ID" or "concept ID" elsewhere. But we have kept the coding system as part of the column name (exactly as you've done here) where there are multiple coding systems in play. So for this specific column, maybe `snomedct_code`?

Reply:
Good to know, thank you. 8b4e506 and b5ee243 update the properties based on `ConceptId`, although I need to investigate that column in more detail.

Reply:
We now have `ctv3_code` and `snomedct_code`. Does ehrQL have a mechanism for communicating that the former shouldn't contain null-values, but the latter could?

Reply:
> Does ehrQL have a mechanism for communicating that the former shouldn't contain null-values, but the latter could?

It does indeed! By default, everything is considered nullable, but a `NotNull` constraint can be applied:
https://github.com/opensafely-core/ehrql/blob/bdd093584a7560cf91d4da586e9a1e13b6362e24/ehrql/tables/beta/core.py#L20-L24

Reply:
Great, thanks Dave. I should have spotted instances of `Constraint`. I've added them in #1296.